Tests/UnitTests/SubscriberAttributes/BackendSubscriberAttributesTests.swift

```diff
@@ -329,7 +329,10 @@ class BackendSubscriberAttributesTests: XCTestCase {
         let eTagManager = MockETagManager(userDefaults: MockUserDefaults())
         self.mockETagManager = eTagManager
 
-        return MockHTTPClient(systemInfo: self.systemInfo, eTagManager: eTagManager, sourceTestFile: file)
+        return MockHTTPClient(systemInfo: self.systemInfo,
+                              eTagManager: eTagManager,
+                              timeoutSeconds: 7,
```

Comment:
why do some use `7` and some `3`? Are they fully arbitrary?

Reply:
I like to keep it arbitrary- I can test that I modified it from the default.

Reply:
Do you mind checking that the total test time didn't go up? I myself introduced some very slow tests a while back that I then fixed by making sure we didn't wait this long in some tests so it would be nice to make sure that didn't regress 🙏🏻

Reply:
This would reduce the time, the default `NSURLSession` timeout is 60 seconds, this sets the full response and request timeout to 7 seconds instead 😄